land-degradation-and-development.csl

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>Land Degradation &amp; Development</title>
+    <title-short>LDD</title-short>
+    <id>http://www.zotero.org/styles/land-degradation-and-development</id>
+    <link href="http://www.zotero.org/styles/land-degradation-and-development" rel="self" xml:lang="en"/>
+    <link href="http://www.zotero.org/styles/international-journal-of-climatology" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)1099-145X/homepage/ForAuthors.html" rel="documentation" xml:lang="en"/>
+    <author>
+      <name>Gilberto Mendes</name>
+      <email>gilbertomendes@ufu.br</email>
+    </author>
+    <contributor>
+      <name>Gilberto Mendes</name>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="science"/>
+    <issn>1085-3278</issn>
+    <eissn>1099-145X</eissn>
+    <summary>The Land Degradation &amp; Development citation style.</summary>
+    <updated>2016-05-13T00:43:49+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " delimiter-precedes-last="never"/>
+      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol" et-al-min="3" et-al-use-first="1" initialize-with=". "/>
+      <et-al font-style="italic"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text variable="container-title" font-style="italic"/>
```

Comment:
this seems odd -- if you don't have the same line in the author macro, matching citation to bibliography would be quite tricky.